code/game/objects/items/food/misc.dm

```diff
@@ -1,39 +1,54 @@
 
 ////////////////////////////////////////////OTHER////////////////////////////////////////////
 
-/obj/item/food/cheesewheel
+
+/obj/item/food/cheese/cheese/wheel
```

Comment:
This type path doesn't look right, shouldn't have `cheese/cheese`!

Reply:
Hm... I must have replaced it when I replaced all food/cheesewheel to food/cheese/wheel